tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -118,9 +119,12 @@ export class WebGPUBackend extends KernelBackend {
   private pipelineCache:
       {[key: string]: GPUComputePipeline|Promise<GPUComputePipeline>};
   private programTimersStack: TimerNode[];
-  private querySet: GPUQuerySet;
+
+  private queryResolveBuffer: GPUBuffer = null;
+  private querySet: GPUQuerySet = null;
+  private querySetCount = 2;
```

Comment:
`querySetCount` is not needed since it's always 2?

Reply:
This needs to be used in 2 functions.